data/sql/updates/pending_db_world/rev_1625248824338191500.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1625248824338191500');
+
+-- Update Kroshius' SAI and make it not selectable
+UPDATE `creature_template` SET `unit_flags` = 33555200, `AIName` = 'SmartAI' WHERE (`entry` = 14467);
+
```

Comment:
Check flags and bits from our wiki https://www.azerothcore.org/wiki/sql-standards

Reply:
Sure, I guess. I just copied the query from Keira3 😄